index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Cooking with Lina</title>
+	<header><h1>Cooking with Lina<h1></header>
+</head>
+<body>
+	<p>Hello, and welcome to my website about cooking!</p>
```

Comment:
I like that you divided up the document using sections, but these p tags should be inside some sectioning element as well.